source/install.txt

```diff
@@ -10,168 +10,390 @@ Install Compass
+|compass-short| and follow the given tutorials.
+
+.. tabs::
+
+   tabs:
+     - id: osx
+       name: macOS
+       content: |
+
+     - id: windows
+       name: Windows
+       content: |
+
+     - id: ubuntu
+       name: Ubuntu
+       content: |
+
+
+     - id: rhel
+       name: RHEL
+       content: |
+
+
+Prerequisites
 -------------
 
-|compass| works with MongoDB Server version 3.0 and higher, both
-Enterprise and Community editions. It runs on 64-bit versions
-of Mac OS X, Windows, Ubuntu 14.04+ and RHEL 7+.
+Software Requirements
+~~~~~~~~~~~~~~~~~~~~~
 
-Software requirements to run |compass-short| are listed below:
+.. tabs::
+
+   hidden: true
+
+   tabs:
+     - id: osx
+       name: macOS
+       content: |
+
+         |compass-short| requires the 64-bit version of macOS 10.10 or
+         later and MongoDB 3.0 or later.
+
+     - id: windows
+       name: Windows
+       content: |
+
+         |compass-short| requires the 64-bit version of Microsoft
+         Windows 7 or later and MongoDB 3.0 or later.
 
-.. include:: /includes/fact-compass-platforms.rst
+         .. note::
 
-Installation
-------------
+            The Windows installation of |compass| requires **Microsoft
+            .NET Framework version 4.5 or later.** The |compass-short|
+            installer prompts you to install the minimum required
+            version of the .NET framework if it is not already installed
+            on your system.
+
+     - id: ubuntu
+       name: Ubuntu
+       content: |
 
-.. note::
+         |compass-short| requires the 64-bit version of Ubuntu 14.04 or
+         later and MongoDB 3.0 or later.
 
-   As of MongoDB Server version 3.6, MongoDB packages a
-   platform-specific |compass-short| installation script with
-   `MongoDB Server downloads
-   <https://www.mongodb.com/download-center/enterprise>`_. For details,
-   see the :manual:`install_compass script
-   </reference/program/install_compass/>` documentation.
+     - id: rhel
+       name: RHEL
+       content: |
+
+         |compass-short| requires the 64-bit version of RHEL 7+ or later
+         and MongoDB 3.0 or later.
+
+Network Requirements
+~~~~~~~~~~~~~~~~~~~~
+
+Certain features of |compass|, such as third-party mapping services and
+Intercom, require inbound connections from external sources to
+function. In some instances, users have reported issues with their
+system firewall blocking these network requests and thereby limiting
+|compass| functionalities. To fix this issue, configure your system
+firewall to allow incoming connections for |compass|.
+
+Ensure ports ``80`` and ``443`` are open between |compass| and the
+host for the MongoDB Instance to which Compass connects.
+
+.. include:: /includes/fact-isolated-edition-firewall.rst
+
+Download Compass
+----------------
+
+.. admonition:: |compass-short| bundled with MongoDB 3.6 and later
+   :class: important
+
+   If you choose to install |compass-short| when you install
```

Comment:
I think we need a contextual sentence here stating that Compass is bundled with server. It's in the admonition title, but I think we should preface this sentence about installation by explaining a bit.

Reply:
Agreed. Added.